### DISCUSSION TITLE
[Question][Hw4] Get # of covered lines and # of unique crashes

### DISCUSSION THREAD

**Discussion by lemonadeeeee** [prosyslab-classroom/cs348-information-security#556] [Q&A]
In the evaluation criteria it says that we're going to be graded based on the # of covered lines and # of unique crashes. I am not completely sure how to get these 2 values automatically. Currently, I am manually checking each crash input in the output directory, but I was wondering if there is an easier (and less tedious) way to get these criteria out from the output directory.

Edit: I wrote a bash script to automate this process
"reproduce the crash with an crashing input"
./example1 < output_dir/crash/input1
but I'm still not entirely sure how to get the # of covered lines.

Edit2: I also wanted to ask if the submission for this hw is the same as hw3 where we only need to push to github, and we don't have to submit anything on klms.

**Comment by RiceBiscuits** [comment]
1. Did you check test directory? You can find *.coverage file under the test directory. You can just count lines of each coverage files.
2. Please submit your HW 3-5 to corresponding Github repository. You don't need to upload any file to KLMS.

**Comment by lemonadeeeee** [comment]
Completely forgot about the .cov files. Thanks for the help
